xc7/utils/prjxray_routing_import.py

```diff
@@ -709,7 +709,7 @@ def create_get_pip_wire_names(conn):
     def get_pip_wire_names(pip_pkey):
         cur.execute(
             """SELECT src_wire_in_tile_pkey, dest_wire_in_tile_pkey
-            FROM pip_in_tile WHERE pkey = ? AND is_pseudo = 0;""",
+            FROM pip_in_tile WHERE pkey = ? AND (is_pseudo = 0 OR name LIKE '%CLK_HROW_CK%');""",
```

Comment:
I actually think this should just be `WHERE pkey = ?`, I think the previous `is_pseudo = 0` was not required given that this function takes a primary key.

Reply:
Done.